upup/pkg/fi/cloudup/openstack/cloud.go

```diff
@@ -496,6 +497,9 @@ func buildLoadBalancerClient(c *openstackCloud, spec *kops.OpenstackSpec, provid
 			return fmt.Errorf("error building lb client: %w", err)
 		}
 		lbClient = client
+		if spec.Loadbalancer.FlavorID != nil {
```

Comment:
hmm is these 3 rows really needed? I do not see where c.flavorID is used?

Reply:
@zetaab you're right: openstackCloud struct should not be influenced here, sorry for the oversight. Can I push another commit reverting the changes to cloud.go back?

Reply:
Just make new commit to remove that

Reply:
done @zetaab